### DISCUSSION TITLE
Hacked account.

### DISCUSSION THREAD

**Discussion by Mrpib1054** [community/community#144957] [New to GitHub]
Hello.My name is Matthew Jacques and a hacker opened an account using my email address and is using my credit card too..I was able to access some of his participation on here and it looks like he is a total failure and owes you money..the username is jax1054@protonmail.com..He also used all my info to open a Microsoft 365 account and has totally hacked my wi go to do Xbox live and I paid for that too..you can contact me at mmj105@yahoo.com ..maybe you could have a name for him or any other information..thank you and I hope to hear from you..Matthew Jacques

**Comment by ebndev** [comment]
I’m sorry that you’re having to go through this, @Mrpib1054 

It sounds like you have been a victim of serious harassment and/or abuse. Though we can’t directly help with your situation, you might be able to receive assistance through [Crash Override Network](http://www.crashoverridenetwork.com/) and [RAINN](https://www.rainn.org/).
